www/front_src/src/route-components/administration/extensions/manager/index.js

```diff
@@ -425,6 +425,12 @@ class ExtensionsRoute extends Component {
       extensionDetails,
     } = this.state;
 
+    const hasNoSelection =
+      ((installed && not_installed && updated) ||
+        (!installed && !not_installed && !updated)) &&
```

Comment:
I don't understand these conditions:
How can we get in a state where `installed` and `not_installed` are both true or false at the same time?

Reply:
That is the same behavior as before.

Reply:
@Thebarda I understand that, I'm just trying to make sense of it now to try to improve it :)

Reply:
I completly about that. That makes no sense because this is not the same behavior as status filter on auto discovery